packages/@webex/plugin-authorization-browser-first-party/src/authorization.js

```diff
@@ -363,34 +402,40 @@ const Authorization = WebexPlugin.extend({
           this.cancelQRCodePolling();
         })
         .catch((res) => {
-          if (this.pollingRequest === null) return;
+          // if the pollingId has changed, it means that the polling request has been canceled
+          if (this.currentPollingId !== this.pollingId) return;
 
-          if (currentAttempts >= maxAttempts) {
-            this.eventEmitter.emit('qRCodeLogin', {
-              eventType: 'authorizationFailure',
-              data: {message: 'Authorization timed out'}
-            });
-            this.cancelQRCodePolling();
+          // When server sends 400 status code with message 'slow_down', it means that last request happened too soon.
+          // So, skip one interval and then poll again.
+          if (res.statusCode === 400 && res.body.message === 'slow_down') {
+            schedulePolling(interval * 2);
```

Comment:
so the server doesn't indicate to us some back-off period through a something like retry-after header or something in the body? it would be better if they did....
Also, shouldn't they use 429 error code for this scenario?

Reply:
Unfortunately the server do not give us any clue
headers:
<img width="735" alt="image" src="https://github.com/user-attachments/assets/b491a80b-f611-4651-8a5f-75bb5d37d15b">

Response:
```json
{
    "message": "slow_down",
    "errors": [
        {
            "description": "Device polling too frequently."
        }
    ],
    "trackingId": "webex-web-client_320f4982-d9e0-4dc2-b713-027c720db466_13"
}
```

But In the first request we get what should be the time interval between the polling requests. 

We get the `slow_down` error because we started the timer for the next request before the last request finished.